Modules/_testcapi/vectorcall_limited.c

```diff
@@ -32,6 +32,101 @@ LimitedVectorCallClass_new(PyTypeObject *tp, PyTypeObject *a, PyTypeObject *kw)
     return self;
 }
 
+static PyObject *call_vectorcall(PyObject* self, PyObject *callable) {
+    PyObject *args[3], *kwname, *kwnames;
+
+    args[0] = NULL;
+    args[1] = PyUnicode_FromString("foo");
+    args[2] = PyUnicode_FromString("bar");
+    kwname = PyUnicode_InternFromString("baz");
+    kwnames = PyTuple_New(1);
+
+    if (!args[1] || !args[2] || !kwname || !kwnames ||
+        PyTuple_SetItem(kwnames, 0, kwname)) {
```

Comment:
Minor PEP 7 nit (ditto for the `if`s below): 

```suggestion
        PyTuple_SetItem(kwnames, 0, kwname))
    {
```